### ISSUE TITLE
No Update?

### ISSUE THREAD

**Issue by JamesCoyle** [silvestreh/atom-material-ui#163]
From the lack of the rewrite branch and updated readme page I'm assuming the rewrite was merged with the master branch. It doesn't seem to be available in atom though: the [Atom IO Package page](https://atom.io/packages/atom-material-ui) is still showing the old version. Is that intended?

**Comment by silvestreh** [comment]
@JamesCoyle yeah, that's intentional. I released the new theme and turned out it was filled with bugs and inconsistencies with other packages. So much so, that I had to unpublish it and revert to the last one.

Right now, the `1.0.x` release only lives in the repository and I'm still looking for testers. It has come a long way since the failed release, but I still want to stress test it as much as possible because there are a couple hundred thousand users and I would like to avoid annoying them (**again**) :grin: 

That said, you're welcome to clone the repo and test the hell out of this theme. There are instructions in the contributing section of the `README.md` file.
